server/bin/clortho

```diff
@@ -135,6 +135,11 @@ app.get('/provision', routes.provision);
 app.get('/node_status', routes.elb_check);
 app.get('/ldap_status', routes.checkStatus);
 
+// Dockerflow
+app.get('/__lbheartbeat__', function(req, resp) { resp.send(200); });
+app.get('/__heartbeat__', routes.checkStatus);
```

Comment:
Do you wanna use this local function definition, or refer to the route in `server/routes` that does the same thing?